core/admin/mailu/utils.py

```diff
@@ -265,7 +268,7 @@ def save(self):
 
         # set uid from dict data
         if self._uid is None:
-            self._uid = self.app.session_config.gen_uid(self.get('user_id', ''))
+            self._uid = self.app.session_config.gen_uid(self.get('_user_id', ''))
```

Comment:
What's going on here?

Reply:
It's not set by us and Flask-login now prepends values with an underscore:

https://github.com/maxcountryman/flask-login/blob/c760c0ef7ccc95d49b4693200245a4f2b148d41b/CHANGES#L32